dmon.c

```diff
@@ -137,6 +138,22 @@ signal_to_name (int signum)
     return unknown;
 }
 
+#if defined(__UCLIBC__)
+static int getloadavg(double *a, int n)
```

Comment:
I would move the `#include <sys/sysinfo.h>` line here, because the function is only used inside the `getloadavg()` fallback implementation for uCLibc.

Reply:
Good idea, done!